manic/checkout.py

```diff
@@ -216,9 +217,10 @@ def commandline_arguments(args=None):
                         '%(prog)s. By default only summary information '
                         'is provided. Use verbose output to see details.')
 
-    parser.add_argument('-v', '--verbose', action='store_true', default=False,
+    parser.add_argument('-v', '--verbose', action='count', default=0,
                         help='Output additional information to '
-                        'the screen and log file.')
+                        'the screen and log file. This flag can be used '
+                        'multiple times.')
```

Comment:
Maybe be more explicit here, like, "This flag can be used up to 2 times, increasing the verbosity level each time."

Reply:
Actually, maybe it only is effective the second time for '--status', in which case this could say, "For --status, this flag can be used a second time, increasing the verbosity level each time."